src/SectionParserData.h

```diff
@@ -51,6 +51,12 @@ namespace snowcrash {
         /** Table mapping named type to sub types */
         mson::NamedTypeInheritanceTable namedTypeInheritanceTable;
 
+        /** Table mapping named types to their dependent named types */
+        mson::NamedTypeDependencyTable namedTypeDependencyTable;
+
+        /** Variable to store the current named type */
+        mson::Literal namedTypeContext;
```

Comment:
Hmm so this is storing the context – name of the named type being parsed – to store dependencies for later?